fapi-client/src/main/scala/com/gu/facia/api/utils/FaciaContentImplicits.scala

```diff
@@ -0,0 +1,119 @@
+package com.gu.facia.api.utils
+
+import com.gu.contentapi.client.model.{Element, Tag, Content}
+import com.gu.facia.api.models.{ImageReplace, ImageCutout, FaciaContent}
+import org.joda.time.DateTime
+
+import scala.util.Try
+
+object FaciaContentImplicits {
+
+  implicit class FaciaContentImplicit(fc: FaciaContent) {
```

Comment:
I don't understand why you'd have both, it's just a burden. Part of good library design is choosing a good usage and then encouraging it. Having multiple ways of doing the same thing is confusing.

The pure way is certainly more flexible and removes the indirection of using implicits, so as you say I think that's the better approach and it would get my vote for the one to keep.